app/views/users/show.html.erb

```diff
@@ -1 +1,39 @@
-<h1>Here are the details of the user</h1>
+<html>
+  <head>
+    <%= stylesheet_link_tag 'users_css' %>
+  </head>
+
+  <body>
+    <header>
+      <h1>Here are the details of the user <%= @user.name %></h1>
+    </header>
+    <main>
+      <section>
+        <%= render 'users/user', user: @user %>
+        <div class="user-bio flex-column">
+          <p class="post-align">Bio:</p>
+          <p class="post-align"><%= @user.bio %></p>
+        </div>
+
+        <h2>This is the List of Posts:</h2>
+        <ul class="post-list flex-column">
+          <% @user.returns_the_three_most_recent_posts.each do |post| %>
+            <li class="post-list-item flex-column">
+              <h3 class="post-id post-align">Post #<%=  link_to post.id, user_post_path(@user, post) %></h3>
+              <h4 class="post-title post-align">Title: <%=  link_to post.title, user_post_path(@user, post) %></h4>
+              <p class="post-text post-align"><%= post.text %></p>
+              <p class="comments-likes-counter counter-align">
+                Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+              </p>
+            </li>
+          <% end %>
+        </ul>
+        <div class="btn flex-column">
+          <button class="see-btn" type="button">
+            <%= link_to "See all posts", user_posts_path(@user) %>
+          </button>
+        </div>
+      </section>
+    </main>
+  </body>
```

Comment:
- [ ] Similarly here, kindly make sure that your design matches the given [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB5V373V756SA6RLQ345YG2QZGC5QPA) ✨💡 Kindly have a look at the comparison below 👇


Current | Given
--- | ---|
![image](https://github.com/oovillagran/blog_app/assets/106119105/0b1da622-57fc-4ee4-abff-a60a71eade94) | ![image](https://github.com/oovillagran/blog_app/assets/106119105/b6fc5021-6d23-4504-935b-b7dfc9a7b196)

Reply:
Done.